sdbuild/packages/usbgadget/motd

```diff
@@ -0,0 +1,3 @@
+[95m************************************************[39m
+[95m*      [31mWelcome to PYNQ on Ultra96      [95m*[39m
```

Comment:
If we want the usbgadget to be Ultra96-specific we should put it in the Ultra96 repo - you can create a `packages` directory under Ultra96 - alternatively we can create the motd at install time based on the BOARD variable